### PR TITLE
[9.0] add crm_meeting_commercial_partner

### PR DESCRIPTION
**Pull request by JordiBForgeFlow**
CRM Meeting Commercial Partner
==============================

This module allows to display the meetings of a company as the sum of all
the meetings held with the company's contacts.

Usage
=====
Go to the partner views. You will see, in the Kanban and Form views, that
the number of meetings shown for a company includes the meetings held with
the company's contacts.

### CODE REVIEW

**Comment by JordiBForgeFlow** [comment]
cc @Eficent

File: crm_meeting_commercial_partner/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+# For copyright and license notices, see __openerp__.py file in root directory
+##############################################################################
```
Comment (LoisRForgeFlow) [inline crm_meeting_commercial_partner/__init__.py:4]: remove these lines

File: crm_meeting_commercial_partner/models/__init__.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+# For copyright and license notices, see __openerp__.py file in root directory
+##############################################################################
```
Comment (LoisRForgeFlow) [inline crm_meeting_commercial_partner/models/__init__.py:4]: remove these lines

File: crm_meeting_commercial_partner/tests/__init__.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+# For copyright and license notices, see __openerp__.py file in root directory
+##############################################################################
```
Comment (LoisRForgeFlow) [inline crm_meeting_commercial_partner/tests/__init__.py:4]: remote these lines

**Review by LoisRForgeFlow — APPROVED** [review]
Functionally works good :+1: 

just some comments about headers in line

**Comment by rafaelbn** [comment]
Hi @jbeficent @lreficent are you still interested in this PR? I see travis red :red_circle:

**Review by claudiagn — APPROVED** [review]
functional review ok! lgtm 👍

**Pull request closed without merging**